src/mixin-lifecycle-methods.js

```diff
@@ -55,6 +55,10 @@ module.exports = function mixinLifecycleMethods(BaseElement = HTMLElement) {
 				return;
 			}
 
+			// Overwrite ... this means that this initialize
+			// can't be inherited (super.initialize).
+			this[inSetupSymbol] = true;
```

Comment:
if initialize is called many times, we need to make sure this keeps getting added ... this means the lifecycle always needs to be "last mixin" AND users can't really overwrite without fixing this symbol.  I wonder if we need `initialized()` for users?